lib/clever-ruby/models/school_admins_response.rb

```diff
@@ -15,20 +15,24 @@
 module Clever
 
   class SchoolAdminsResponse
-    attr_accessor :data
+    attr_accessor :data, :paging, :links
 
 
     # Attribute mapping from ruby-style variable name to JSON key.
     def self.attribute_map
       {
-        :'data' => :'data'
+        :'data' => :'data',
+        :'paging' => :'paging',
+        :'links' => :'links'
       }
     end
 
     # Attribute type mapping.
     def self.swagger_types
       {
-        :'data' => :'Array<SchoolAdminResponse>'
+        :'data' => :'Array<SchoolAdminResponse>',
+        :'paging'=> :'Object',
+        :'links'=> :'Object'
```

Comment:
A couple minor style notes: 

* the quotes are unnecessary unless there are special characters in the symbol
* prefer using the shorthand `{ foo: :bar }` over the hashrocket style where possible
* spacing will need to get fixed on line 34-35

Reply:
@OwaisRaza1994 , lets update it, per Steve, please! It's ok, we can merge it tomorrow.

Reply:
oh well, it's merged now :) 
you can add it later...